pkg/plugins/runtime/k8s/controllers/pod_controller.go

```diff
@@ -334,6 +338,7 @@ func (r *PodReconciler) createOrUpdateDataplane(
 		},
 	}
 	operationResult, err := kube_controllerutil.CreateOrUpdate(ctx, r.Client, dataplane, func() error {
+		r.Log.Info("UPDATING DATAPLANE", "dataplane", dataplane)
```

Comment:
judging by the upper case it's probably a leftover?

Reply:
it was debug log, removed